Source/DafnyLanguageServer.Test/Lookup/HoverVerificationTest.cs

```diff
@@ -194,7 +195,10 @@ method Abs(x: int) returns (y: int)
 
     [TestMethod, Timeout(5 * MaxTestExecutionTimeMs)]
     public async Task IndicateSlowHints() {
```

Comment:
I don't understand this test. I looked for a 'slow hint' in the expected message but I don't see one.

Reply:
I changed the test name.